neuro_san/internals/network_providers/agent_network_storage.py

```diff
@@ -71,10 +71,10 @@ def add_agent_network(self, agent_name: str, agent_network: AgentNetwork):
         for listener in self.listeners:
             if is_new:
                 listener.agent_added(agent_name, self)
-                self.logger.info("ADDED network for agent %s", agent_name)
+                self.logger.info("ADDED network for agent %s : %d", agent_name, id(agent_network))
             else:
                 listener.agent_modified(agent_name, self)
-                self.logger.info("REPLACED network for agent %s", agent_name)
+                self.logger.info("REPLACED network for agent %s : %d", agent_name, id(agent_network))
 
```

Comment:
Add debug information to logging - id() of AgentNetwork instance to identify it.